iothub/device/src/AmqpTransportSettings.cs

```diff
@@ -17,6 +17,7 @@ public sealed class AmqpTransportSettings : ITransportSettings
         private readonly TransportType _transportType;
         private TimeSpan _operationTimeout;
         private TimeSpan _openTimeout;
+        internal string AuthenticationChain { get; set; }
```

Comment:
Add a comment here explaining the usage, and also indicating that this is specific to IoT Edge implementation.